purchase_control_menu/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module recovers the Purchase control menu as it was in the former version.
```

Comment:
could you make add a more descriptive text. For newcomers in V13 (and then V14...) it doesn't mean much.

Reply:
Right, I developed it a little more